integration-test/1955-chinese-parser.py

```diff
@@ -62,6 +206,132 @@ def test_san_francisco_osm(self):
             16, 10482, 25330, 'places',
             {'name:zh-default': u'旧金山/三藩市/舊金山'})
 
+        self.generate_fixtures(dsl.way(26819236, wkt_loads(
```

Comment:
My last 2¢: Please remove this fixture and later 2 tests as I think they duplicate the `test_united_states` which starts on line 16, and that test has the corrected coordinates while this one still tests for zoom 16 in error.

Reply:
that makes sense, I removed in the next commit.